File: Starters/Starters 50/MILEAGE/Solution.c
#include <stdio.h>

int main(){
    int t;
    scanf("%d", &t);
    for (int i = 0; i < t; i++){
        float n, x, y, a, b;
        scanf("%f%f%f%f%f", &n, &x, &y, &a, &b);
        float pet = x/a;
        float die = y/b;
        if (pet<die) printf("Petrol\n");
        else if (pet>die) printf("Diesel\n");
        else printf("Any\n");   
    }
}

File: Starters/Starters 50/MILEAGE/Solution.go
package main
import "fmt"

func main(){
  var t,n,x,y,a,b int
  var pet, die float32
  fmt.Scanln(&t)
  for i:=0;i<t;i++ {
    fmt.Scan(&n,&x,&y,&a,&b)
    pet = x/a
    die = y/b
    if pet < die {
      fmt.Println("PETROL")
    } else if pet > die {
      fmt.Println("DIESEL")
    } else {
      fmt.Println("ANY")
    }
  }
}
